### ISSUE TITLE
porechop does not install with gcc 4.8.5 envs

### ISSUE THREAD

**Issue by erinyoung** [artic-network/artic-ncov2019#17]
My default gcc for my OS is 4.8.5. This matters because when I went to use the yaml to create the conda environment for the artic protocol, artic's version of porechop did not install. I didn't realize that 1) artic was using their specific version of porechop, and 2) that it didn't install because there was no error message. 

When I went to install artic's version of porechop in the environment, I came across this error:
`error: [Errno 2] No such file or directory: 'porechop/cpp_functions.so'`

I now knew that it was a compiling issue (needs >gcc 5) and added this to my environment:

```
conda install -c omgarcia gcc-6
pip3 install git+https://github.com/artic-network/Porechop.git@v0.3.2pre
```

Additionally, I realized that bcftools=1.9 and seqtk=1.3 didn't install in the conda environment either. I didn't realize this until after I went through each dependency after I realized the porechop issue.

It'd be helpful if there was a version check for each required package.

**Comment by Koen-vdl** [comment]
Hi, Just wanted to say I noticed the same issue with porechop. Your fix didn't do the trick for me though. 
I ended up using the manual python3 install, which worked:

```
git clone https://github.com/artic-network/Porechop.git
cd Porechop
python3 setup.py install
```

In the end I discovered that the pipeline doesn't use porechop anymore (my consensus sequences are identical with / without porechop in my PATH). The pipeline now uses Guppy barcoder for demuxing. So I guess this issue is not relevant any more.
